appinventor/appengine/src/com/google/appinventor/server/UserInfoServiceImpl.java

```diff
@@ -73,6 +76,11 @@ public Config getSystemConfig(String sessionId) {
 
     // Check to see if we need to upgrade this user's project to GCS
     storageIo.checkUpgrade(userInfoProvider.getUserId());
+
+    // Fetch list of allowed tutorial prefixes from the data store
+    List<String> urls = storageIo.getTutorialsUrlAllowed();
```

Comment:
We could just pass the result of `getTutorialsUrlAllowed` to `setTutorialUrlAllowed` rather than storing it in a variable here.

Reply:
It's like this because when I was testing I had added a code to display the URLs in the log, so I could verify all was well. I then removed the debugging code. I left it this way because it is more readable, it is clear for example that we are dealing with a list of URLs, say as opposed to a string or something else. I did verify that the compiler optimizes this away. So from a performance standpoint, it doesn't hurt at all (I ran a decompiler on the resultant class file).

So, I would prefer to leave it as is.

Reply:
Works for me.